doc/release-notes-1.9.rst

```diff
@@ -2,6 +2,42 @@
 
 .. _zephyr_1.9:
 
+Zephyr Kernel 1.9.2
+###################
+
+This is a maintenance release with fixes.
+
+Kernel
+******
+* Generic queue item acquisition fixed to always return a valid item when
+  using K_FOREVER
+
+Bluetooth
+*********
+* Multiple stability fixes for BLE Mesh
+* Multiple stability fixes for the BLE Controller
+
+Zephyr Kernel 1.9.1
+###################
+
+This is a maintenance release with fixes and a two new features in the
```

Comment:
Since we're not including Jira/GitHub issues being fixed here, how about:

     This is a maintenance release with fixes and two new features in the
     BLE Controller, summarized below.  For more details, see the `GitHub 1.9.1 release page`_.

     .. _GitHub 1.9.1 release page:
     https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v1.9.1